plugins/scheme-tpm-enacttrust/test/cmd/gen-token/gen-key.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+# Copyright 2022 Contributors to the Veraison project.
+# SPDX-License-Identifier: Apache-2.0
+if [[ "" == "$1" ]]; then
+    echo "Usage: gen-key.sh OUTFILE"
+    exit 1
+fi
+set -x -e
+openssl genpkey -algorithm EC -out $1 -pkeyopt ec_paramgen_curve:P-256 #-pkeyopt ec_param_enc:explicit
```

Comment:
should we drop the commented out code?

```suggestion
openssl genpkey -algorithm EC -out $1 -pkeyopt ec_paramgen_curve:P-256
```